.github/workflows/sys-bindings-generator.yml

```diff
@@ -32,20 +36,58 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - uses: actions/setup-go@v4
         with:
-          go-version: '>=1.18'
+          go-version: '>=1.20'
       - name: Install current Bash on macOS
         if: runner.os == 'macOS'
         run: brew install bash coreutils
       - name: No-prefix build for ${{ matrix.os }}
         env:
           AWS_LC_SYS_NO_PREFIX: "1"
         run: |
-          cargo test -p aws-lc-sys --features bindgen,ssl
+          cargo test -p aws-lc-sys --features bindgen
       - name: Collect symbols
         run: |
           ./scripts/build/collect_symbols.sh -c aws-lc-sys
       - name: Commit & Push changes
         run: ./scripts/ci/ci_add_commit_rebase_push.sh "Symbols from ${{ matrix.os }}"
+  collect-windows-symbols-and-commit:
```

Comment:
In order for our CI tests to confirm these changes properly, it needs the artifacts generated and included on the PR branch. For review purposes, I often put link directly to the "manual commits" in the description so that it's more clear what changes require the most scrutiny.

This is an unusual PR b/c it's making substantial changes to our automated bindings generation mechanism. Typically we only run this mechanism with 2-3 line "version bump" commit immediately prior to a new release.